sktime/forecasting/pytorchforecasting.py

```diff
@@ -222,6 +295,79 @@ class PytorchForecastingNBeats(_PytorchForecastingAdapter):
+    model_path: string (default=None)
+        try to load a existing model without fitting. Calling the fit function is
+        still needed, but no real fitting will be performed.
+    random_log_path: bool (default=False)
+        use random root directory for logging. This parameter is for CI test in
+        Github action, not designed for end users.
+
+    Examples
+    --------
+    >>> # import packages
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> from sktime.forecasting.pytorchforecasting import PytorchForecastingNBeats
+    >>> from sktime.utils._testing.hierarchical import _make_hierarchical
+    >>> # generate random data
+    >>> data = _make_hierarchical(
+    ...     hierarchy_levels=(5, 200), max_timepoints=50, min_timepoints=50, n_columns=3
+    ... )
+    >>> # define forecast horizon
+    >>> max_prediction_length = 5
+    >>> fh = ForecastingHorizon(range(1, max_prediction_length + 1), is_relative=True)
+    >>> # split y data for train and test
+    >>> l1 = data.index.get_level_values(1).map(lambda x: int(x[3:]))
+    >>> y_train = data.loc[l1 < 190, "c2"].to_frame()
+    >>> y_test = data.loc[l1 >= 180, "c2"].to_frame()
+    >>> len_levels = len(y_test.index.names)
+    >>> y_test = y_test.groupby(level=list(range(len_levels - 1))).apply(
+    ...     lambda x: x.droplevel(list(range(len_levels - 1))).iloc[:-max_prediction_length]
+    ... )
+    >>> # define the model
+    >>> model = PytorchForecastingNBeats(
+    ...     trainer_params={
+    ...         "max_epochs": 5,  # for quick test
+    ...         "limit_train_batches": 10,  # for quick test
+    ...     },
+    ... )
+    >>> # fit and predict
+    >>> model.fit(y=y_train, fh=fh) # doctest skip
+    PytorchForecastingNBeats(trainer_params={'limit_train_batches': 10,
+                                            'max_epochs': 5})
+    >>> y_pred = model.predict(fh, y=y_test)
+    >>> print(y_test)
+                                c2
+    h0   h1     time
+    h0_0 h1_180 2000-01-01  6.308914
+                2000-01-02  3.471440
+                2000-01-03  4.169305
+                2000-01-04  5.990554
+                2000-01-05  5.611347
+    ...                          ...
+    h0_4 h1_199 2000-02-10  6.448248
+                2000-02-11  4.290731
+                2000-02-12  5.494657
+                2000-02-13  4.752948
+                2000-02-14  5.243385
+
+    [4500 rows x 1 columns]
+    >>> print(y_pred)
+                                c2
+    h0   h1     time
+    h0_0 h1_180 2000-02-15  5.167375
+                2000-02-16  5.178759
+                2000-02-17  5.251082
+                2000-02-18  5.331861
+                2000-02-19  5.372994
+    ...                          ...
+    h0_4 h1_199 2000-02-15  5.005799
+                2000-02-16  4.998720
+                2000-02-17  5.031197
+                2000-02-18  5.081184
+                2000-02-19  5.113482
+
+    [500 rows x 1 columns]
+    >>>
```

Comment:
why are these three `>>>` here. If not required by dostring, please remove them. This applies to all of the examples.

Reply:
removed